kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-scenario-simulation/pom.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <groupId>org.kie.server</groupId>
+    <artifactId>kie-server-rest</artifactId>
+    <version>7.31.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>kie-server-rest-scenario-simulation</artifactId>
+
+  <name>KIE :: Execution Server :: Remote :: REST :: Scenario Simulation</name>
+  <description>KIE Server Scenario Simulation Extension</description>
+
+  <properties>
+    <java.module.name>org.kie.server.rest.scenariosimulation</java.module.name>
+  </properties>
+
+  <dependencies>
```

Comment:
There seems to be unused dependencies:
```
[WARNING] Unused declared dependencies found:
[WARNING]    org.apache.commons:commons-lang3:jar:3.8.1:compile
[WARNING]    org.kie.soup:kie-soup-dataset-api:jar:7.31.0-SNAPSHOT:compile
[WARNING]    org.kie:kie-internal:jar:7.31.0-SNAPSHOT:compile
[WARNING]    org.mockito:mockito-core:jar:1.10.19:test
```

Reply:
Good catch, done (not able to find `mockito-core` so maybe it is a false positive)